dev_apps/music-nga/test/marionette/lib/music.js

```diff
@@ -0,0 +1,493 @@
+/* global require, module */
+'use strict';
+
+var assert = require('assert');
+
+function Music(client, origin) {
+  this.client = client;
+  this.origin = origin || ('app://' + Music.DEFAULT_ORIGIN);
+  this.actions = client.loader.getActions();
+}
+
+module.exports = Music;
+
+Music.DEFAULT_ORIGIN = 'music-nga.gaiamobile.org';
+
+Music.Selector = Object.freeze({
+  viewFrame: '#view-stack iframe',
```

Comment:
Good point. I have an activeViewFrame somewhere in another subsequent branch that use ```iframe.active``` to explicitely pick the active one, which could be the secondary.